HidingNavigationBar/HidingNavigationBarManager.swift

```diff
@@ -216,6 +219,7 @@ open class HidingNavigationBarManager: NSObject, UIScrollViewDelegate, UIGesture
 		default:
 			break;
 		}
+		handleScrolling()
```

Comment:
@tristanhimmelman  This was a bug fix, where the nav bar position gets messed up, when coming back from control center or background, What this taken care of ?

Reply:
I missed this! Sorry about that. I am working to include this and you iOS11 handling of insets.

Reply:
Sure, Thanks 👍